eve/main.yml

```diff
@@ -948,6 +928,39 @@ stages:
           command: >
             tox -e tests -- -m "${PYTEST_FILTERS}"
           haltOnFailure: true
+      - ShellCommand:
+          name: Copy Cypress tests
+          env:
+            SSH_CONFIG: >-
+              eve/workers/openstack-single-node-rhel/terraform/ssh_config
+          command: >
+            scp -r -F "$SSH_CONFIG" ui bootstrap:
```

Comment:
Since we're running these tests in a Terraform env, it would really make sense to add an extra VM for running these tests outside of the cluster. This would avoid Cypress eating resources the underlying tested cluster needs.